grades/models.py

```diff
@@ -146,3 +150,59 @@ def create_pending(cls, course_run):
             raise CourseRunGradingAlreadyCompleteError(
                 'Course Run "{0}" has already been completed'.format(course_run.edx_course_key))
         return course_fg_info
+
+
+class ProctoredExamGrade(TimestampedModel, AuditableModel):
+    """
+    Model to store proctored exam grades (like the pearson exams)
+    """
+    # relationship to other models
+    user = models.ForeignKey(User, null=False)
+    course = models.ForeignKey(Course, null=False)
+
+    # fields from proctorate exam results
+    passing_score = models.FloatField()
+    score = models.FloatField()
+    grade = models.TextField()
+    client_authorization_id = models.TextField()
+
+    # dump of all the remaining fields coming from the proctorate exam results
+    row_data = JSONField(null=None)
+
+    # fields coming from unknown source but about proctorate exam
+    # we need the following to compute the percentage grade
+    maximum_exam_score = models.FloatField()
+
+    # custom fields based on proctorate exam results
+    passed = models.BooleanField(null=False)
```

Comment:
This is redundant since `BooleanField` doesn't allow `null=True`, they require `NullBooleanField` instead for that: https://docs.djangoproject.com/en/1.10/ref/models/fields/#null